loan-payoff/REPWRITERSPECS/BANNO.LOAN.PAYOFF.V1.POW

```diff
@@ -633,11 +647,31 @@ PROCEDURE READCONFIGFILESETTINGS
           TERMSLASTLINEUSED=TERMSLINESCOUNT
         END
 
-      [ Print config info]
+[* Print config info. This is a hidden parameter setting which can override the default Print
+** config info setting. Use this "PC:" parameter setting in the CFG Letter file with "ON"
```

Comment:
Good catch, Robby. I've added it to what is checked, but changed the working to talk about "Yes", since people may think about that when seeing a parameter called "Print config info".  The change will be in my next commit, hopefully today.